codespell_lib/data/dictionary.txt

```diff
@@ -664,13 +667,15 @@ allowd->allowed
 allowe->allowed, allow,
 allpication->application
 allready->already
+allright->all right
```

Comment:
This should probably also suggest alright:
https://en.oxforddictionaries.com/usage/all-right-or-alright

Reply:
I removed it.

Reply:
Please put it back in, I think we're all agreed that allright is a typo, so it should remain, but there are two acceptable corrections, so both need listing.

Reply:
done.